styles/stylesheet.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background-color: rgb(236,236,236);
+  align-items: center;
+  margin: 1rem;
+}
+
+.dogface {
+  height: 280px;
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  border-color: rgb(29, 71, 152);
+}
+.header {
+  position: relative;
+  margin: 5px;
+  width: auto;
+}
+.logo {
+  position: relative;
+  margin: 5px;
+  width: 33%;
+}
+.buttons {
+  font-family: inherit;
+  font-size: 100%;
+  font-color: rgb(9,60,96);
+  text-align: center;
+  box-sizing: 100%;
+  border-style: solid;
+  border-color: rgb( 236,236,236);
+  background-color: rgb( 236,236,236);
+  display: flex;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: rgb(236, 236,236);
+  width: 100%;
+  position: absolute;
+  margin-top: 45px;
+  }
+
+.red-shirt {
+  float:left;
+  width: 25%;
+  margin-left: 10px;
+}
+
+.img {
+      height: 200px;
+      width: inherit;
+}
+
+.input [type=submit] {
+  width:100%;
+  background-color: rgb(129,183,65);
+  border-radius: 15px;
+}
+.adopt{
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  justify-content: space-around;
+  align-content: center;
+  background-color: white;
+  margin-top: 70px;
+  margin-left: 5px;
+  margin-right:5px;
+}
+
+.box {
+  height:100%;
+  width:20%;
+  border: 4px solid rgb(236,236,236);
+  background-color: white;
+  opacity: .8;
+  padding: 14px 16px;
+}
+.dog {
+  width: 100%;
+  height: 20%;
+}
+.blog {
+  padding-top: 5px;
+  background-color: white;
+  width: 100%
+  float: left;
+  justify-content: space-around;
+  position: relative;
+  margin-top:110px;
+  text-align: left;
+  padding-bottom: 150px;
+}
+.blogpic {
+  width: 20%;
+  float: left;
+  position: relative;
+  justify-content: space-around;
+}
+.Travel {
```

Comment:
All of these blog posts can be combined to one css class and use that same css class for each of the blogs